torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14601,7 +14601,8 @@ def ref_pairwise_distance(input1, input2):
         supports_forward_ad=True,
         supports_fwgrad_bwgrad=True,
         supports_out=False,
-        sample_inputs_func=partial(sample_inputs_dropout, train=True, min_input_dim=2),
+        # As per the docs, valid input dims are (4, 5)
+        sample_inputs_func=partial(sample_inputs_dropout, train=True, valid_input_dim=(4, 5)),
```

Comment:
This is OpInfo entry for `feature_alpha_dropout`.

Reply:
This is good for the purposes of this PR, but eventually `FeatureAlphaDropout` should be documented and tested to work on dims besides 4 and 5 (see https://github.com/pytorch/pytorch/issues/69960#issuecomment-995169300)